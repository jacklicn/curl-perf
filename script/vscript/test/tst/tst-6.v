
var res

// TODO: compiler - if function returns a value that is not used, then release it.

sub foo( a, b, c)
    print( 123 )
    print( a + b )
    print( a )
    print( b )
    print( c )

    return c
end

res = foo( 1, 2, 3 + 4 * 5 )


