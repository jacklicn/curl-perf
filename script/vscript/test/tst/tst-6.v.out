*** syntax ok ***
*** semantics and code generation ok ***

>123

>3

>1

>2

>23
